app/code/Magento/Version/Test/Unit/Controller/Index/IndexTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Version\Test\Unit\Controller\Index;
+
+use Magento\Version\Controller\Index\Index as VersionIndex;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\ProductMetadataInterface;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+
+/**
+ * Class \Magento\Version\Test\Unit\Controller\Index\IndexTest
+ */
+class IndexTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var VersionIndex
+     */
+    private $model;
+
+    /**
+     * @var Context
+     */
+    private $context;
+
+    /**
+     * @var ProductMetadataInterface
+     */
+    private $productMetadata;
+
+    /**
+     * @var ResponseInterface
+     */
+    private $response;
+
+    /**
+     * Prepare test preconditions
+     */
+    protected function setUp()
+    {
+        $this->context = $this->getMockBuilder(Context::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->productMetadata = $this->getMockBuilder(ProductMetadataInterface::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getName', 'getEdition', 'getVersion'])
+            ->getMock();
+
+        $this->response = $this->getMockBuilder(ResponseInterface::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['setBody', 'sendResponse'])
+            ->getMock();
+
+        $this->context->expects($this->any())
+            ->method('getResponse')
+            ->willReturn($this->response);
+
+        $helper = new ObjectManager($this);
+
+        $this->model = $helper->getObject(
```

Comment:
Instantiate the controller by `new`. You are passing mocked arguments so using object manager is inapropriate here

Reply:
The funny part is the fact, that was merged already :rofl: